uppsrc/CtrlCore/X11DHCtrl.cpp

```diff
@@ -274,28 +274,29 @@ void DHCtrl::Terminate(void)
 
 	// Unmaps the window
 	MapWindow(false);
-
+	if(utop) {
```

Comment:
GetTop() is more secure than checking utop only.

Reply:
Probably, yes. I am trying to trace that bool top used in Ctrl.h and all those 
And I do not like
` bool Ctrl::IsWndOpen() const {
	GuiLock __;
//	DLOG("IsWndOpen " << Upp::Name(this) << ": " << (bool)top);
	return top;
} `
And the main thing that bool topmost also exists...
I would suggest maybe to decide different concepts and different naming. 
and maybe to rename istop or isopen or ismapped